media/css/privacy/privacy.less

```diff
@@ -193,7 +193,7 @@
       background-position: center -40px;
     }
 
-    &.policy-firefox-rocket a:before {
+    &.policy-firefox-lite a:before {
```

Comment:
The image sprite that this selector references still contains the old rocket logo, is that expected? I don't see where this is used.

Reply:
This is really just in prep for this being added to the sidebar of the main privacy page. It's not there yet, so we've got some time to update this. Honestly using a sprite is no longer advantageous since we have HTTP2 enabled, but that's beside the point :) We should fix up the sprite though, you're right. Do you know if that's part of another issue or should that be fixed in this one?

Reply:
Yeah, this page is old and so is the sprite. I'd say don't worry about it right now, as it's not used in the page. We can update the logo later on, and maybe even get rid of the sprite while we're at it?